### PR TITLE
ICL Enchanced 

### PR DESCRIPTION
**Pull request by Itskiprotich**

### CODE REVIEW

**Comment by esthermmoturi** [comment]
Thanks @Itskiprotich

File: IntelliSOFT/Example CHT application/cht_pvers/test/forms/assessment.spec.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ describe('Assessment form test', () => {
     beforeEach(async () => {
         await harness.clear();
         // set harnes date to Jan 1st 2023
-        return await harness.setNow('2023-08-24');
+        return await harness.setNow('2024-04-03');
```
Comment (IreneAchola) [inline IntelliSOFT/Example CHT application/cht_pvers/test/forms/assessment.spec.js]: Is there a particular reason why the date is set (and changed) to this date? Also, the comment above is misleading.

File: IntelliSOFT/Example CHT application/cht_pvers/test/forms/chw_follow.spec.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ describe('CHP Follow Up form test', () => {
     beforeEach(async () => {
         await harness.clear();
         // set harnes date to Jan 1st 2023
-        return await harness.setNow('2023-08-24');
+        return await harness.setNow('2024-04-03');
```
Comment (IreneAchola) [inline IntelliSOFT/Example CHT application/cht_pvers/test/forms/chw_follow.spec.js]: ^^ ditto

File: IntelliSOFT/Example CHT application/cht_pvers/test/forms/death_confirmation.spec.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ describe('Death Confirmation form test', () => {
     beforeEach(async () => {
         await harness.clear();
         // set harnes date to Jan 1st 2023
-        return await harness.setNow('2023-08-24');
+        return await harness.setNow('2024-04-03');
```
Comment (IreneAchola) [inline IntelliSOFT/Example CHT application/cht_pvers/test/forms/death_confirmation.spec.js]: ^^ ditto

File: IntelliSOFT/Example CHT application/cht_pvers/test/forms/padr.spec.js [hunk 4]
```diff
@@ -14,7 +14,7 @@ describe('PADR form test', () => {
     beforeEach(async () => {
         await harness.clear();
         // set harnes date to Jan 1st 2023
-        return await harness.setNow('2023-08-24');
+        return await harness.setNow('2024-04-03');
```
Comment (IreneAchola) [inline IntelliSOFT/Example CHT application/cht_pvers/test/forms/padr.spec.js]: ^^ ditto

File: IntelliSOFT/Example CHT application/cht_pvers/test/forms/follow.spec.js [hunk 5]
```diff
@@ -14,7 +14,7 @@ describe('CHW Supervisor Confirmation form test', () => {
     beforeEach(async () => {
         await harness.clear();
         // set harnes date to Jan 1st 2023
-        return await harness.setNow('2023-08-24');
+        return await harness.setNow('2024-04-03');
```
Comment (IreneAchola) [inline IntelliSOFT/Example CHT application/cht_pvers/test/forms/follow.spec.js]: ^^ ditto

**Review by IreneAchola — COMMENTED** [review]
I've left some inline comments

File: IntelliSOFT/Example CHT application/cht_pvers/test/forms/assessment.spec.js (hunk 1 above)
Comment (Itskiprotich) [reply IntelliSOFT/Example CHT application/cht_pvers/test/forms/assessment.spec.js]: This has been update to pick current date by default.

**Review by esthermmoturi — APPROVED** [review]

**Pull request merged**